mkt/webapps/serializers.py

```diff
@@ -468,9 +456,10 @@ def fake_object(self, data):
         # regions stored in ES instead of making SQL queries.
         obj.get_excluded_region_ids = lambda: data['region_exclusions']
 
-        # Set up payments stuff to avoid extra queries.
+        # Set up payments stuff to avoid extra queries later (we'll still make
+        # some, because price info is not in ES).
         if obj.is_premium():
-            self._attach_payments_info(obj)
+            Webapp.attach_premiums([obj])
```

Comment:
Doing this higher up, in `field_to_native()`/data`, would spare us a few queries when there are multiple premium apps returned, I might do that later once all tests pass.